js/humanize/decimal.js

```diff
@@ -0,0 +1,156 @@
+define(['component/ko-validation/config'], function (config) {
+    'use strict';
+
+    function getParts(input, decimalChar) {
+        input = '' + input;
+        var fracIndex = input.lastIndexOf(decimalChar) + 1;
+        var parts = {
+            negative: input.substring(0, 1) == '-' ? '-' : '',
+            fractional: fracIndex ? input.substr(fracIndex).replace(/\D/g, '') : '',
+            integral: (fracIndex ? input.substring(0, fracIndex) : input).replace(/\D/g, '')
+        };
+        parts.toFloat = function toFloat() {
+            return parseFloat(parts.negative + '0' + parts.integral + '.' + parts.fractional);
+        };
+        return parts;
+    }
+
+    function getFormat() {
+      return {
```

Comment:
@ljqx I sign off, and will merge the change, please take care of the node in next iteration.

Thanks
Wei